politeiad/cmd/legacypoliteia/legacypoliteia.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 The Decred developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+)
+
+const (
+	// Command names. See the usage.go file for details on command usage.
+	convertCmdName = "convert"
+	importCmdName  = "import"
+
+	// serverPubkey is the former politeia public key from when it ran the git
+	// backend.
+	serverPubkey = "a70134196c3cdf3f85f8af6abaa38c15feb7bccf5e6d3db6212358363465e502"
+)
+
+func _main() error {
```

Comment:
I would like a CLI flag that allows me to specify a single token whose contents will be converted and saved to disk so that I can test and verify one proposal at a time.